packages/rpc-graphql/src/schema/instruction.ts

```diff
@@ -873,718 +874,736 @@ export const instructionTypeDefs = /* GraphQL */ `
+                            return 'AssignWithSeedInstruction';
+                        }
+                        if (instruction.instructionType === 'transferWithSeed') {
+                            return 'TransferWithSeedInstruction';
+                        }
+                    }
+                    if (instruction.programName === 'vote') {
+                        if (instruction.instructionType === 'initialize') {
+                            return 'VoteInitializeAccountInstruction';
+                        }
+                        if (instruction.instructionType === 'authorize') {
+                            return 'VoteAuthorizeInstruction';
+                        }
+                        if (instruction.instructionType === 'authorizeWithSeed') {
+                            return 'VoteAuthorizeWithSeedInstruction';
+                        }
+                        if (instruction.instructionType === 'authorizeCheckedWithSeed') {
+                            return 'VoteAuthorizeCheckedWithSeedInstruction';
+                        }
+                        if (instruction.instructionType === 'vote') {
+                            return 'VoteVoteInstruction';
+                        }
+                        if (instruction.instructionType === 'updatevotestate') {
+                            return 'VoteUpdateVoteStateInstruction';
+                        }
+                        if (instruction.instructionType === 'updatevotestateswitch') {
+                            return 'VoteUpdateVoteStateSwitchInstruction';
+                        }
+                        if (instruction.instructionType === 'compactupdatevotestate') {
+                            return 'VoteCompactUpdateVoteStateInstruction';
+                        }
+                        if (instruction.instructionType === 'compactupdatevotestateswitch') {
+                            return 'VoteCompactUpdateVoteStateSwitchInstruction';
+                        }
+                        if (instruction.instructionType === 'withdraw') {
+                            return 'VoteWithdrawInstruction';
+                        }
+                        if (instruction.instructionType === 'updateValidatorIdentity') {
+                            return 'VoteUpdateValidatorIdentityInstruction';
+                        }
+                        if (instruction.instructionType === 'updateCommission') {
+                            return 'VoteUpdateCommissionInstruction';
+                        }
+                        if (instruction.instructionType === 'voteSwitch') {
+                            return 'VoteVoteSwitchInstruction';
+                        }
+                        if (instruction.instructionType === 'authorizeChecked') {
+                            return 'VoteAuthorizeCheckedInstruction';
+                        }
+                    }
+                    return resolveCustomInstructionType(instruction);
```

Comment:
Same here, this whole `__resolveType(..)` block didn't change, I just added this injectable resolve function at the end of the conditional.